### ISSUE TITLE
atmosphere have built-in incongito 

### ISSUE THREAD

**Issue by ELY3M** [jimzrt/Incognito_RCM#27]
I think it is time to archive this repo.   
remove the releases....

**Comment by ELY3M** [comment]
I am closing my repo too.

**Comment by deejay87** [comment]
hi, yes ok i have read this but this setup only work if you boot fusee-primary and don't work with launch hekate, if it's that it's not very the best solution
Using the rcm work with all

**Comment by ELY3M** [comment]
thats why I recommend people to use fuse-priminary after they are done with emummc setup.

**Comment by deejay87** [comment]
yes, the new hekate can do this but for the moment have a bug, i have open an issue :)

**Comment by paprika714** [comment]
Sorry to bother you, what do you mean with Atmosphere having a build in incognito?

I've just installed a fresh version of Atmosphere, what should I do?

**Comment by ELY3M** [comment]
read this  https://switch.hacked.us/extras/incognito/

**Comment by paprika714** [comment]
> read this https://switch.hacked.us/extras/incognito/

Thank you for the link, really appreciated.
Can I still use FTP programs to communicate via WiFi with my computer or it will block completely internet access ?

**Comment by dbenoy** [comment]
This tool will still be needed forever in order to restore backups made in the past. I'm currently in that situation myself. I'd like to use Atmosphere's built-in incognito, but I need to restore my backup first. Sadly I'm being blocked from being able to do so by #22.

**Comment by dbenoy** [comment]
Also you may want to boot OFW when you're not using homebrew, but you may be afraid that traces of CFW activity may exist on your device.

**Comment by ELY3M** [comment]
hmm I guess I need to update my repo and make new bin...  but I will remove the ability to incoginto...   only use it to restore from your prodinfo backup

**Comment by ELY3M** [comment]
wifi and internet should not be affected at all.

**Comment by technobyteongithub** [comment]
Maybe don't archive the repo so that others can use **Incognito RCM** on older versions?

**Comment by bao3** [comment]
No, many people run CFW(sysNAND), EMUMMC, Stock  , these 3 os . if we use AMS to set incognito , then CFW(sysNAND) could not connect to the nintendo . But many people like me run CFW(sysNAND) to play online games .

**Comment by technobyteongithub** [comment]
> No, many people run CFW(sysNAND), EMUMMC, Stock , these 3 os . if we use AMS to set incognito , then CFW(sysNAND) could not connect to the nintendo . But many people like me run CFW(sysNAND) to play online games .

@bao3 I don't quite understand what your saying XD Is CFW(sysNAND) a copy of the system to the sd card? if yes whats EMUMMC? and how do you play online games??? Sorry I'm new to the switch RCM stuff.

**Comment by bao3** [comment]
> > No, many people run CFW(sysNAND), EMUMMC, Stock , these 3 os . if we use AMS to set incognito , then CFW(sysNAND) could not connect to the nintendo . But many people like me run CFW(sysNAND) to play online games .
> 
> @bao3 I don't quite understand what your saying XD Is CFW(sysNAND) a copy of the system to the sd card? if yes whats EMUMMC? and how do you play online games??? Sorry I'm new to the switch RCM stuff.

CFW(sysNAND) , is a Stock OFW , but you could run cheats, and all custom software . EmuMMC is a clone of the Stock OFW, which is set to the SDcard.
